src-electron/importexport/import-json.js

```diff
@@ -438,6 +439,29 @@ async function jsonDataLoader(db, state, sessionId, packageMatch) {
         pkg.type,
         pkg.version
       )
+      // Checking if standalone xml file content changed by checking for a crc
+      // mismatch. If there is a crc mismatch then delete the package so that
+      // it does not come under the list of existing packages.
+      // This check makes sure that if a custom xml is updated and the zap file
+      // is re-opened then the custom xml is reloaded to reflect the correct
+      // configuration.
+      if (pkg.type == 'zcl-xml-standalone') {
+        let info = await queryPackage.getPackageByPackageId(db, packageId)
```

Comment:
I thought we have a function for this somewhere. util.readFileContentsAndCrc() or something?